{
	"id": "SUSE-SU-2023:2970-1",
	"modified": "2023-07-25T13:55:17Z",
	"published": "2023-07-25T13:55:17Z",
	"related": [
		"CVE-2023-25399",
		"CVE-2023-29824"
	],
	"summary": "Security update for python-scipy",
	"details": "This update for python-scipy fixes the following issues:\n\n- CVE-2023-25399: Fixed minor refcounting issue in Py_FindObjects (bsc#1213062).\n- CVE-2023-29824: Fixed use-after-free in Py_FindObjects (bsc#1213137).\n",
	"affected": [
		{
			"package": {
				"ecosystem": "openSUSE:Leap 15.4",
				"name": "python-scipy",
				"purl": "purl:rpm/suse/python-scipy&distro=openSUSE%20Leap%2015.4"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "1.2.0-150100.4.6.1"
						}
					]
				}
			],
			"ecosystem_specific": {
				"binaries": [
					{
						"python3-scipy_1_2_0-gnu-hpc": "1.2.0-150100.4.6.1"
					}
				]
			}
		},
		{
			"package": {
				"ecosystem": "openSUSE:Leap 15.4",
				"name": "python-scipy_1_2_0-gnu-hpc",
				"purl": "purl:rpm/suse/python-scipy_1_2_0-gnu-hpc&distro=openSUSE%20Leap%2015.4"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "1.2.0-150100.4.6.1"
						}
					]
				}
			],
			"ecosystem_specific": {
				"binaries": [
					{
						"python3-scipy_1_2_0-gnu-hpc": "1.2.0-150100.4.6.1"
					}
				]
			}
		},
		{
			"package": {
				"ecosystem": "openSUSE:Leap 15.5",
				"name": "python-scipy_1_2_0-gnu-hpc",
				"purl": "purl:rpm/suse/python-scipy_1_2_0-gnu-hpc&distro=openSUSE%20Leap%2015.5"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "1.2.0-150100.4.6.1"
						}
					]
				}
			],
			"ecosystem_specific": {
				"binaries": [
					{
						"python3-scipy_1_2_0-gnu-hpc": "1.2.0-150100.4.6.1"
					}
				]
			}
		}
	],
	"references": [
		{
			"type": "ADVISORY",
			"url": "https://www.suse.com/support/update/announcement/2023/suse-su-20232970-1/"
		},
		{
			"type": "REPORT",
			"url": "https://bugzilla.suse.com/1213062"
		},
		{
			"type": "REPORT",
			"url": "https://bugzilla.suse.com/1213137"
		},
		{
			"type": "WEB",
			"url": "https://www.suse.com/security/cve/CVE-2023-25399"
		},
		{
			"type": "WEB",
			"url": "https://www.suse.com/security/cve/CVE-2023-29824"
		}
	]
}
